### ISSUE TITLE
Support for nested tables

### ISSUE THREAD

**Issue by tobiaskn** [asciidoctor/asciidoctor-pdf#413]
This seems to be related to #6 but is specific for nested tables.

When embedding a table within a table cell (using the AsciiDoc style for the column) it renders just fine as   HTML but in PDF the inner table is output as raw text.

Example:

```
[cols="d,a",options="header"]
|===
| Name
| Additional Information

| Flags
|
// Nested table
.Bit Field
[cols="d,d",options="header"]
!===
! Bit
! Name

! 1
! Entry valid

! 2
! Date valid

!===
|===
```

Would the current work on #6 also lead to support for nested tables or has it to be tackled separately?

**Comment by mojavelinux** [comment]
I want to say yes, solving #6 will pave the way to solving this problem, but that's just an educated guess. However, this is low priority atm given the other problems we still need to solve. My hope is that we sort of "get it for free" by fixing #6 properly. If not, it's going to be hard for me to justify working on it.

**Comment by mojavelinux** [comment]
This was implemented when #6 was implemented.
